plugins/CoreAdminHome/Commands/ResetInvalidations.php

```diff
@@ -0,0 +1,203 @@
+        );
+
+        $this->addNoValueOption(
+            'dry-run',
+            null,
+            'Perform a dry run without making changes. Shows which invalidations would be reset without actually modifying them.'
+        );
+
+        $this->setHelp(
+            'This command allows administrators to reset stuck invalidations that are incorrectly marked as "in progress". '
+            . 'This can happen if an archiving process was interrupted, such as during a server crash or deployment, leaving '
+            . 'invalidations in a stuck state. Resetting them ensures they can be reprocessed in the next archiving run.
+
+Usage examples:
+
+- Reset all stuck invalidations for site ID 1 that were started more than an hour ago:
+  `./console core:reset-invalidations --idsite=1 --older-than="1 hour ago"`
+
+- Reset invalidations assigned to a specific host:
+  `./console core:reset-invalidations --processing-host=archiver1.example.com`
+
+- Perform a dry run to check which invalidations would be reset:
+  `./console core:reset-invalidations --idsite=1 --older-than="1 hour ago" --dry-run`
+
+- Reset invalidations for multiple sites and hosts:
+  `./console core:reset-invalidations --idsite=1 --idsite=10 --processing-host=archiver1 --processing-host=archiver2`
+
+Use this command with caution, especially when resetting invalidations while archiving processes are still in progress.'
+        );
+    }
+
+    protected function doExecute(): int
+    {
+        $dryRun = $this->getInput()->getOption('dry-run');
+
+        $whereCondition = $this->generateWhereCondition();
+
+        if ($dryRun) {
+            $rows = Db::fetchAll(
+                'SELECT name, idsite, report, date1, date2, period, ts_invalidated, ts_started, processing_host, process_id FROM '
+                . Common::prefixTable('archive_invalidations') . ' WHERE ' . $whereCondition['sql']
+                . ' ORDER BY ts_started',
+                $whereCondition['bind']
+            );
+
+            if (count($rows) === 0) {
+                $this->getOutput()->writeln('No invalidations found.');
+            } else {
+                $this->getOutput()->writeln(count($rows) . ' invalidations found:');
+                if (count($rows) > 50) {
+                    $rows = array_slice($rows, 0, 50);
+                    $this->getOutput()->writeln('Output limited to oldest 50 records');
+                }
+                $this->renderTable(array_keys($rows[0]), $rows);
+            }
+        } else {
+            $queryObj = Db::query(
+                'UPDATE ' . Common::prefixTable('archive_invalidations')
+                    . ' SET status = 0, processing_host = NULL, process_id = NULL, ts_started = NULL WHERE '
+                    . $whereCondition['sql'],
+                $whereCondition['bind']
+            );
+            $rowCount = $queryObj->rowCount();
+
+            $this->getOutput()->writeln($rowCount . ' invalidations have been reset.');
```

Comment:
I usually get an eye twitch when I see 1 of something with plural noun, e.g. '1 invalidations'. Do we have an established system how to deal with numbers and messaging, correctly distinguishing zero, one and multiple items? There's `ngettext`, but not a single use in our codebase.

I guess alternatively we can adjust the output to something like `Number of invalidations that were reset: 1`. Then it can be 0, 1, 15 etc. and it always works.

Not a blocker, just a thing it might be good to start thinking of in general.

```suggestion
            $this->getOutput()->writeln('Number of invalidations that were reset: ' . $rowCount);
```